content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -1,11 +1,27 @@
 +++
 title = "Installation"
-description = "Options for deploying Kubeflow Pipelines"
 weight = 1
 +++
 
 {{% kfp-v2-keywords %}}
 
-This page will be available soon. For similar information, see [KFP v1 installation documentation][v1-installation].
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
+[Kubeflow deployment](/docs/started/#installing-kubeflow), you also have a choice
+to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize manifests.
 
-[v1-installation]: /docs/components/pipelines/legacy-v1/installation/
\ No newline at end of file
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
+
+## Deploying Kubeflow Pipelines
```

Comment:
Please format this more like the ["Model Registry" page](https://www.kubeflow.org/docs/components/model-registry/installation/) with two sections:

1. Prerequisites
1. Installing Kubeflow Pipelines:
     - Installing on Kubeflow Platform
          - Explain what this means, and the differences to using standalone.
          - Core differences to standalone include "multi user support", "connecting to external object stores", "connecting to external MySQL databases", etc.
    - Standalone Installation:
         - The steps you have already listed.
3. Next Steps

Reply:
I don't have the bandwidth to add instructions for all of this at the moment. I did what I could offer. I agree we need more information on this page. Are you willing to send a PR?